1.hello-world/README.md

```diff
@@ -121,15 +125,27 @@ You're up and running! Both Dapr and your app logs will appear here.
 
 ## Step 4 - Post Messages to your Service
 
-Now that Dapr and our Node.js app are running, let's POST messages against it. **Note**: here we're POSTing against port 3500 - if you used a different port, be sure to update your URL accordingly.
+Now that Dapr and our Node.js app are running, let's POST messages against it, using different tools. **Note**: here we're POSTing against port 3500 - if you used a different port, be sure to update your URL accordingly.
+
+First, let's POST the message by using Dapr cli in a new command line terminal:
+
+Windows
+```sh
+dapr send --app-id nodeapp --method neworder --payload '{\"data\": { \"orderId\": \"41\" } }'
```

Comment:
`send` command will be changed to `invoke` in the next release. please make sure that we will change this in the next release
cc/ @yaron2 @msfussell

Reply:
I know. I did not change it yet to keep it consistent with 0.2. Thanks for the reminder.